Installing-and-Configuring/BootOptions.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: default
+title: Configuring Boot Options
+permalink: /BootOptions/
+parent: Installing and configuring
+nav_order: 85
+---
+
+<!-- markdownlint-disable MD033 -->
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+<!-- markdownlint-enable MD033 -->
+
+
+Boot config files
+===
+
+A user has the option to make persistent modifications to the non-Qubes boot
+ process by creating one or more of the following files:
+
+| file | description |
+| ---- | ---- |
+| kexec_menu.txt | contains multiple options for parameters to the kexec command|
+| kexec_hashes.txt | a sha256sum file from within the respective boot directory |
+| kexec_iso_add.txt | a sh variable to override the standard ISO kernel argument
+ additions |
+| kexec_iso_remove.txt | a sh variable to override the standard ISO kernel
+ argument removals |
+| kexec_default.$N.txt | specifies the default kexec parameters corresponding to
+ the Nth menu option |
+| kexec_default_hashes.txt | a sha256sum file for the default entry kexec file
+ parameters |
+| kexec_rollback.txt | a sha256sum of the TPM counter contents in the tmp
+ directory |
+| kexec_key_devices.txt | contains a list of "device uuid" combos for all LUKS
+ devices to unlock |
+| kexec_key_lvm.txt | contains the name of an LVM group to activate on boot |
+
+These can be placed in any of the following locations:
+
+| location | description |
+| ---- | ---- |
+| /boot/ | used during internal HD boot |
+| /media/ | used during standard USB boot |
+| /media/kexec_iso/$ISO_FILENAME/ | used during USB boot from a particular ISO
+ file |
+
+These files are only used if there is an appropriate signature for them in
+ `kexec.sig` covering all `kexec*.txt` in that location.  This can be generated
+  by running `kexec-sign-config -p /boot/`, etc.  These files are copied by
+ `kexec-check-config` to `/tmp/kexec/` only there's a valid signature.  From
+  there the boot routines reference only the configs in `/tmp/kexec`.
+
+
+Dynamic vs Persistent Boot Options
+====
+
+There are two ways for heads to boot Operating systems from /boot.
+
+* Dynamic (no kexec_menu.txt)
+* Persistent
+
+If there is no persistent `kexec_menu.txt`, the boot directory will be searched
+ for grub/syslinux-like configurations and it will be generated dynamicly (for
+ any of the HD/USB/USB-ISO locations).  Creating a persistent
+ `kexec_menu.txt` can be useful to limit the options displayed or to make
+ persistent alterations to xen or kernel params.
```

Comment:
If there is no persistent `kexec_menu.txt` generated from GUI menu option `Default boot` while /boot content detached signed digest is verified as valid, the boot directory will be searched
 for grub/syslinux-like configurations and it will be generated dynamically (for
 any of the HD/USB/USB-ISO locations).  Creating a persistent
 `kexec_menu.txt` can be useful to limit the options displayed or to make
 persistent alterations to xen or kernel params.

Reply:
incorporated new content but separated into two sentences.  Will push in a few minutes.